packages/eslint-plugin/src/rules/explicit-member-accessibility.ts

```diff
@@ -103,26 +112,6 @@ export default util.createRule<Options, MessageIds>({
     const propCheck = overrides.properties ?? baseCheck;
     const paramPropCheck = overrides.parameterProperties ?? baseCheck;
     const ignoredMethodNames = new Set(option.ignoredMethodNames ?? []);
-    /**
-     * Generates the report for rule violations
-     */
-    function reportIssue(
```

Comment:
I didn't see the point of the wrapper function. It was barely doing anything at all (and made it harder to tweak some calls to use `suggest` instead of `fix`).

IMO, 

```ts
context.report({
   node: methodDefinition,
   messageId: 'unwantedPublicAccessibility',
   data: {
     type: nodeType,
     name: methodName,
   },
   fix: getUnwantedPublicAccessibilityFixer(methodDefinition),
 });
```
is much more readable than
```ts
 reportIssue(
   'unwantedPublicAccessibility',
   nodeType,
   methodDefinition,
   methodName,
   getUnwantedPublicAccessibilityFixer(methodDefinition),
 );
```

and only a _tiny_ bit more verbose.

What do you think?

Reply:
But can we switch the latter to this?

```ts
 reportIssue(
   'unwantedPublicAccessibility',
   nodeType,
   methodDefinition,
   methodName,
);
```

Reply:
Oh, I see, there's both `getUnwantedPublicAccessibilityFixer` and `getMissingAccessibilitySuggestions`. Maybe you could pass the function in as a parameter? But yeah I wouldn't sweat it too much, if you don't think that's cleaner. I can take a deeper look and try to get something on the re-review.

Reply:
The same helper was used for both message ids, so the internal logic of the helper function would have to switch on the message id and add a different fixer. I can do this if you like, but it doesn't seem like a big cleanup to me (and I prefer labeled arguments anyway, for functions like this with 4+ parameters).

Reply:
Yeah, I'm going sour on the whole idea. 😄